tutorials/20_particles_tutorial.md

```diff
@@ -0,0 +1,94 @@
+\page particles Particles
+
+This example shows how to include a particle emitter in the scene.
+
+## Compile and run the example
+
+Create a build directory and use `cmake` and `make` to compile the code:
+
+```{.sh}
+cd ign-rendering/examples/particles_demo
+mkdir build
+cd build
+cmake ..
+make
+```
+Execute the example:
+
+```{.sh}
+./particles_demo
+[Msg] Loading plugin [ignition-rendering4-ogre]
```

Comment:
this should be `ogre2`? i.e. `ignition-rendering4-ogre2`

Reply:
Yup, fixed in 3a233f6.